src/AppBundle/Mail/Sender.php

```diff
@@ -14,36 +15,47 @@ class Sender
     /** @var \Swift_Mailer */
     protected $mailer;
 
+    /** var \Symfony\Bundle\TwigBundle\TwigEngine */
+    protected $templating;
+
+    /** @var string */
+    protected $senderEmail;
+
+    /** @var string */
+    protected $recipientEmail;
+
     /**
      * @param \Swift_Mailer $mailer
      */
-    public function __construct(Swift_Mailer $mailer)
+    public function __construct(
+        Swift_Mailer $mailer,
+        Templating $templating,
+        $senderEmail,
+        $recipientEmail
+    )
     {
         $this->mailer = $mailer;
+        $this->templating = $templating;
+        $this->senderEmail = $senderEmail;
+        $this->recipientEmail = $recipientEmail;
     }
 
     /**
-     * Sends e-mail with content based on form settings.
-     *
-     * @param \AppBundle\Entity\Message $message
-     * @param string $title
-     * @param string $senderEmail
-     * @param string $recipientEmail
-     *
-     * @return bool
+     * @param Contact $contact
      */
-    public function send(Message $message, $title, $senderEmail, $recipientEmail)
+    public function send(Contact $contact)
     {
-        $msg = Swift_Message::newInstance($title, $message->getBody());
-
-        $msg->setFrom($senderEmail)
-            ->setTo($recipientEmail)
-            ->setReplyTo($recipientEmail);
+        $title = 'You have a new message from ' . $contact->getFrom(); //todo handle this with translation : waiting for translation story
```

Comment:
@clash82 I didn't set up translation in this PR cause I want to refactor this too.

I've created a jira for explaining that https://jira.ez.no/browse/DEMO-47.

I think it's better to do it in one PR with all as we already have non translated content in the demo ?